rate/static/html/studio_view.html

```diff
@@ -0,0 +1,30 @@
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="settings-tab">
+  <ul class="list-input settings-list">
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" for="freeform">Freeform prompt</label>
+        <input class="input setting-input" name="freeform" id="freeform" value="{freeform}" type="text" />
+      </div>
+      <span class="tip setting-help">Example: Please provide us feedback on this section.</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" for="likert">Likert prompt</label>
```

Comment:
I can see how this would be confusing to developers. I think in this case, most of the audience (professors -- who are professional researchers) will be familiar with the terminology. If not, I'd rather bring them into the proper terminology with documentation than reinvent terminology.